.github/workflows/main.yml

```
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    container:
+      image: elementary/docker:unstable
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Install Dependencies
+      run: |
+        apt update
+        apt install -y libgconf2-dev libpolkit-gobject-1-dev libswitchboard-2.0-dev elementary-sdk
```

```suggestion
        apt install -y libswitchboard-2.0-dev elementary-sdk
```

We no longer depend on gconf and polkit actually; README stays old :sweat_smile: